pkg/distributor/ha_tracker.go

```diff
@@ -113,9 +120,14 @@ func (cfg *HATrackerConfig) RegisterFlags(f *flag.FlagSet) {
 func newClusterTracker(cfg HATrackerConfig) (*haTracker, error) {
 	codec := codec.Proto{Factory: ProtoReplicaDescFactory}
 
-	if cfg.FailoverTimeout <= cfg.UpdateTimeout {
-		return nil, fmt.Errorf("HA Tracker failover timeout must be greater than update timeout, %d is <= %d", cfg.FailoverTimeout, cfg.UpdateTimeout)
+	if cfg.UpdateTimeoutJitterMax != 0 {
+		cfg.UpdateTimeoutJitter = time.Duration(rand.Int63n(int64(2*cfg.UpdateTimeoutJitterMax))) - cfg.UpdateTimeoutJitterMax
```

Comment:
Right, this field shouldn't be part of `cfg`, but rather `haTracker`. Maybe something like `updateTimeoutJitter := ..` and then use the variable in L136 as part of `t := haTrcker{}``